### PR TITLE
chore: Allow code to be manually deployed from specific branch

### PR DESCRIPTION
**Pull request by vinnyjth**

### CODE REVIEW

**Comment by redreceipt** [comment]
this will most likely fail every time because our build numbers are driven by number of commits

**Comment by vinnyjth** [comment]
@redreceipt I'm okay if the app build fails, I really want to be able to deploy to heroku.

**Comment by vinnyjth** [comment]
I went ahead and split the deploys steps apart :)

**Comment by redreceipt** [comment]
`git push heroku master` should do it?

I think the deploy settings need to stay set to “Heroku Git” for our deploy to work automatically

File: .github/workflows/deploy_client.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Deploy
+name: Deploy Client
```
Comment (redreceipt) [inline .github/workflows/deploy_client.yml:1]:
```suggestion
name: Deploy Mobile App
```

**Comment by vinnyjth** [comment]
@redreceipt I tried that. Doesn't work since our API code is in a sub folder. Our Github action accounts for that.

image

**Review by redreceipt — APPROVED** [review]

**Pull request merged**